src/dra/menu.c

```diff
@@ -66,42 +66,42 @@ const char* D_800A2D18[] = {
 
 #if defined(VERSION_US)
 const char* D_800A2D68[] = {
-    "\x00\x01\x01\xFF",
```

Comment:
It seems like something is off with the ones in this file, there's not a single entry that's coherent. This function doesn't seem to be doing the same encoding? https://github.com/Xeeynamo/sotn-decomp/blob/a4a556239ecab52a7c309fb70fdafe4dde30c4d1/src/dra/menu.c#L930

Reply:
That's certainly possible. Let me see if I can get more context on this.

Reply:
You can see in that function it's the 12x16 font
```
    const int ChWidth = 12;
    const int ChHeight = 16;
```
However these strings aren't shift-jis. I'm guessing they resize some bios font characters and make a table at runtime?